src/main/java/gregtech/integration/jei/recipe/GTRecipeWrapper.java

```diff
@@ -111,11 +111,37 @@ public void drawInfo(Minecraft minecraft, int recipeWidth, int recipeHeight, int
         minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.total", Math.abs((long) recipe.getEUt()) * recipe.getDuration()), 0, 70, 0x111111);
         minecraft.fontRenderer.drawString(I18n.format(recipe.getEUt() >= 0 ? "gregtech.recipe.eu" : "gregtech.recipe.eu_inverted", Math.abs(recipe.getEUt())), 0, 80, 0x111111);
         minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.duration", recipe.getDuration() / 20f), 0, 90, 0x111111);
-        int baseYPosition = 100;
+        minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.min_tier", getMinTierForVoltage(recipe.getEUt())), 0, 100, 0x111111);
```

Comment:
Did you test it on all machines, are you sure that you are not clipping in any of them? Does this cover even Multiblocks?

Reply:
I am probably not familiar enough with gregtech to test this properly on anything with special logic, but it seems to be correct for the electrolyzer.

Reply:
Okay then I will specify it more. Please check these machines that they dos not show any misleading values or it does not cover machine GUI or is covered by it or anything that looks bad: Steam Turbine (representative of energy generators), Assembly machine (problematic GUI), Electric Blast Furnace (multiblock with more lines of informations in GUI).

Reply:
It looks like the assembler already clipped the existing text
![image](https://user-images.githubusercontent.com/1056270/85389257-1b12b780-b53f-11ea-8356-07bbe09eeb24.png)

This line doesn't show at all on steam turbines / diesel generator as they use a different wrapper.

It does however cause clipping on the blast furnace
![image](https://user-images.githubusercontent.com/1056270/85389575-82306c00-b53f-11ea-8185-e4a1ceb715a9.png)

Reply:
@LAGIdiot Any ideas on how to go about sorting this?

Reply:
I was going to look at and properly test it tomorrow as I may be able to get some time for it.

But if you need opinion regarding this right now I can tell you strait of bat that Blast Furnace GUI must be fixed as it does not look good and it may interfere with recipes underneath. Also don't worry about Assembler fluid slot as it is not part of scope, but I can't see on provided picture that added text it is not implemented for that machine? 

And I think that min tier line should be moved under usage maybe even before. @pyure what is your take on that?

And last thing, please change as minimal parts of code as possible because we need to retest everything that is changed.